tests/unittests/OperatorTest.cpp

```diff
@@ -6057,13 +6057,335 @@ TEST_P(OperatorTest, Fp16Splat) {
   }
 }
 
+// Deconvolution with easy to calculate output values. 2x2 input, 1-ch input,
+// 2-ch output.
+TEST_P(OperatorTest, sanityDeconvolution2OutCh) {
+  ENABLED_BACKENDS("Interpreter", "CPU");
```

Comment:
Can we use `CHECK_IF_ENABLED()`? Here and all others. Requires updates to blacklists.